templates/default/tmpl/layout.tmpl

```diff
@@ -29,7 +29,7 @@
 <br class="clear">
 
 <footer>
-    Documentation generated by <a href="https://github.com/jsdoc3/jsdoc">JSDoc <?js= env.version.number ?></a> on <?js= (new Date()) ?>
+    Documentation generated by <a href="https://github.com/jsdoc3/jsdoc">JSDoc<?js if(env.conf.templates.includeDate == undefined || env.conf.templates.includeDate) { ?><?js= env.version.number ?></a> on <?js= (new Date()) ?><?js } ?>
```

Comment:
> This code should be looking for the property `env.conf.templates.default.includeDate`, because this option is specific to the default template.

Indeed. I missed the default object you explained on the issue.

> For safety, check for `env.conf.templates` and `env.conf.templates.default` first.

Included.

> Please check whether the property `!== false`, rather than for `== undefined` or truthy.

Fixed, checking for false simplifies the condition.

> If you want to remove the JSDoc version number as well, please use a separate includeVersion property to control that behavior. (And please make sure the </a> tag shows up when this property is set to false. In your current patch, setting includeDate to false causes the </a> tag to be omitted.)

My bad. I shouldn't be removing the version as well. Fixed.